.github/workflows/ci.yml

```
@@ -39,14 +39,24 @@ jobs:
           lake -R exe cache get # download cache of mathlib docs.
           lake -R build AliveScaling
 
-      - name: Compile Executable 🧐
+      - name: Compile `mlirnatural`Executable 🧐
```

whitespace? A

---

@tobiasgrosser for things like this, it's often easier for everyone if you press  <kbd>⌘</kbd> + <kbd> g </kbd> when writing the review, so you can write down a suggested change, as I've done here: 
```suggestion
      - name: Compile `mlirnatural` Executable 🧐
```